rllib/algorithms/a2c/tests/test_a2c.py

```diff
@@ -10,17 +11,26 @@
 
 
 class TestA2C(unittest.TestCase):
-    """Sanity tests for A2C exec impl."""
 
-    def setUp(self):
-        ray.init(num_cpus=4)
+    num_gpus = float(os.environ.get("RLLIB_NUM_GPUS", "0"))
 
-    def tearDown(self):
+    @classmethod
+    def setUpClass(cls):
+        ray.init(num_cpus=4 if not cls.num_gpus else None)
```

Comment:
Can we not set the num_cpus at all? When the cluster is already setup, what's the point of this? for example I won't be able to run this test as is on a workspace. It will error out saying you cannot initialize a ray cluster with 4 cpus, or sth like that.